helpers/data-formatter.ts

```diff
@@ -0,0 +1,12 @@
+export const getDate = (date: string): string => {
+    const created = new Date(date);
+    return `${created.toLocaleDateString('en-US', { month: 'short' })} ${created.getDay()}, 
+    ${created.getFullYear()}`;
+};
+
+export const printTags = (tags: string[]): string[] => {
```

Comment:
What this functions does?

Reply:
pringTag is a function that helps to format the array tag of each post in an ordered string. However, as there is a problem with the field is in graphql, I am not using it.